.gitignore

```diff
@@ -320,3 +321,7 @@ prefab/
 pythonenv3.8/
 .venv/
 
+###################
+# hpcc
+###################
+.env
```

Comment:
Is this file being used locally for the build.sh script?  I see it ignored here but exported there.

Reply:
Yes - its simply storing the secrets for NuGet and DockerHub (which can also be supplied as args to the script)

Reply:
```
GITHUB_ACTOR=XXX
GITHUB_TOKEN=YYY
DOCKER_USERNAME=AAA
DOCKER_PASSWORD=BBB
```